dot/import_integration_test.go

```diff
@@ -93,21 +93,31 @@ func TestImportState_Integration(t *testing.T) {
 	config := DefaultTestWestendDevConfig(t)
 
 	genFile := NewTestGenesisRawFile(t, config)
-
 	config.ChainSpec = genFile
 	err := InitNode(config)
 	require.NoError(t, err)
 
 	stateFP := setupStateFile(t)
 	headerFP := setupHeaderFile(t)
 
-	const firstSlot = uint64(262493679)
-	err = ImportState(config.BasePath, stateFP, headerFP, trie.V0, firstSlot)
+	genesisBABEConfig := &types.BabeConfiguration{
+		SlotDuration:       1000,
+		EpochLength:        200,
+		C1:                 1,
+		C2:                 4,
+		GenesisAuthorities: []types.AuthorityRaw{},
+		Randomness:         [32]byte{},
+		SecondarySlots:     0,
+	}
```

Comment:
this value repeats a lot throughout the codebase, we could create something like `testGenesisBabeConfig` and reuse that. 
I am not asking to reuse it everywhere. Some amount of redundancy is actually desirable in test data. So, we could try to reuse it when inside the same package.